docs/source/index.rst

```diff
@@ -116,18 +116,22 @@ in `standalone mode <https://docs.luxonis.com/projects/api/en/latest/tutorials/s
    :header-rows: 1
 
    * - OAK-1 PoE
+     - OAK-1 W PoE
      - OAK-D PoE
      - OAK-D W PoE
      - OAK-D Pro W PoE
    * - .. image:: /_static/images/thumbnails/OAK-1-PoE_th.png
           :target: pages/SJ2096POE
+     - .. image:: /_static/images/thumbnails/OAK-1-PoE_th.png
```

Comment:
Perhaps we could refactor, so to have 4 devices in each (PoE devices) table, instead of having 3 in the upper table, and 5 in the lower one?